test/serializers/adapter_for_test.rb

```diff
@@ -20,7 +22,7 @@ def test_overwrite_adapter_with_symbol
         adapter = ActiveModel::Serializer.adapter
         assert_equal ActiveModel::Serializer::Adapter::Null, adapter
       ensure
-
+        ActiveModel::Serializer.config.adapter = @previous_adapter
```

Comment:
Why `ensure` just here? I think you added in order to be able to test [this](https://github.com/rails-api/active_model_serializers/pull/1017/files#diff-a217107028e8b59eb12f0e5077b76509R110)
But if that is the case, we probably should also need to ensure the value of it on other tests there are also changing `ActiveModel::Serializer.config.adapter`.
Maybe wrap this kind of tests into some method defined here.